scripts/src/workflowtesting/checkprforci.py

```diff
@@ -19,10 +19,14 @@ def check_if_ci_only_is_modified(api_url):
     pattern_workflow = re.compile(r".github/workflows/.*")
     pattern_script = re.compile(r"scripts/.*")
     pattern_test = re.compile(r"tests/.*")
+    pattern_release = re.compile(r"release/release_info.json")
+    pattern_readme = re.compile(r"README.md")
+    pattern_docs = re.compile(r"docs/([\w-]+)\.md")
```

Comment:
I would prefer putting these patterns in an array and iterating on them instead of numerating, e.g. `workflow_files = [r".github/workflows/.*", r"scripts/.*", r"tests/.*"]` and `do_not_build_files = [r"release/release_info.json", r"README.md", r"docs/([\w-]+)\.md"]`. Then iterate on them in the while loop instead of repeating. Please adjust the names to your preference since I couldn't come up with a good name.